TobogganApp.ui/src/Components/Card/SHoppingCartCard.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```

Comment:
Getting an error `Cannot find file: 'ShoppingCartCard.js' does not match the corresponding name on disk: '.\src\Components\Card\SHoppingCartCard.js'.`

Reply:
Sorry that error was actually coming from `./src/Views/CartView.js`